src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,63 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import racingcar.domain.car.Car;
+
+public class Cars {
+
+    public static final String NEWLINE = "\n";
+    private final List<Car> cars;
```

Comment:
```suggestion
    public static final String NEWLINE = System.lineSeperator();
    
    private final List<Car> cars;
```

유닉스 계열에서는 "\n"이지만 윈도우 계열에서는 "\r\n"가 기본인데요.
이러한 문제를 해결하기 위해 System.lineSeparator()라는 기능을 지원하고 있습니다!

또한 클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다.